### ISSUE TITLE
afterToggle does not fire first time when startOpen true

### ISSUE THREAD

**Issue by christopherbauer** [jedfoster/Readmore.js#198]
Hi Jed,

I'm seeing a weird issue - when startOpen is enabled for some reason line #269  does not behave itself - 

```
  $element.on('transitionend', (function(_this) {
        return function() {                                           //line 269
          if (_this.options.afterToggle && typeof _this.options.afterToggle === 'function') {
            _this.options.afterToggle(trigger, $element, expanded);
          }

          $(this).attr({
            'aria-expanded': expanded
          }).off('transitionend');
        }
      })(this));
```

If you put a console.log prior to that "return" on line 269 it pops up, however it will not actually execute the afterToggle function on the first collapse. Every subsequent click works just fine.

I'm not too familiar with the usage here of returning a function instead of defining a function for the call - can you shed some light?

**Comment by christopherbauer** [comment]
I am able to circumvent the issue by adding the following line:


`$(readmoreElementSelector).trigger("transitionend");` //this is a shim

**Comment by jedfoster** [comment]
This will be fixed in 3.0.
